server/src/main/java/org/elasticsearch/index/analysis/MultiplexingTokenFilterFactory.java

```diff
@@ -0,0 +1,177 @@
+package org.elasticsearch.index.analysis;
```

Comment:
I think we should move this token filter and its tests to the `org.elasticsearch.analysis.common` package in the `analysis-common` module with all the other migrated analysis components.

Instead of wiring this token filter up in `AnalysisRegistry` you will need to do that in `CommonAnalysisPlugin`.